project/src/main/ui/overworld-ui.gd

```diff
@@ -166,19 +166,6 @@ func _refresh_rect_size() -> void:
 	$Control.rect_size = $Control.get_viewport_rect().size
 
 
-"""
-Returns the level id (if any) corresponding to the curently focused chattable.
-
-This is relevant when the player talks to a creature with a food speech bubble.
-"""
-func _focused_chattable_level_id() -> String:
-	var focused_creature: Creature = ChattableManager.focused_chattable as Creature
-	if not focused_creature:
-		return ""
-	
-	return LevelLibrary.first_unfinished_level_id_for_creature(focused_creature.creature_id)
-
-
```

Comment:
Document this move.

Reply:
Expanded commit comment; `Moved
focused_chattable_level_id utility method from OverworldUi to
ChattableManager. Added focused_chattable_creature_id utility method in
ChattableManager.`